evmcore/dummy_block.go

```diff
@@ -176,24 +177,25 @@ type EvmBlockJson struct {
 
 func (h *EvmHeader) ToJson(receipts types.Receipts) *EvmHeaderJson {
 	enc := &EvmHeaderJson{
-		Number:        (*hexutil.Big)(h.Number),
-		Miner:         h.Coinbase,
-		GasLimit:      0xffffffffffff, // don't use h.GasLimit (too much bits) here to avoid parsing issues
-		GasUsed:       hexutil.Uint64(h.GasUsed),
-		Root:          h.Root,
-		TxHash:        h.TxHash,
-		ParentHash:    h.ParentHash,
-		UncleHash:     types.EmptyUncleHash,
-		Time:          hexutil.Uint64(h.Time.Unix()),
-		TimeNano:      hexutil.Uint64(h.Time),
-		BaseFee:       (*hexutil.Big)(h.BaseFee),
-		Difficulty:    new(hexutil.Big),
-		PrevRandao:    h.PrevRandao,
-		TotalDiff:     new(hexutil.Big),
-		Hash:          &h.Hash,
-		Epoch:         hexutil.Uint64(hash.Event(h.Hash).Epoch()),
-		BlobGasUsed:   (*hexutil.Uint64)(new(uint64)),
-		ExcessBlobGas: (*hexutil.Uint64)(new(uint64)),
+		Number:          (*hexutil.Big)(h.Number),
+		Miner:           h.Coinbase,
+		GasLimit:        0xffffffffffff, // don't use h.GasLimit (too much bits) here to avoid parsing issues
+		GasUsed:         hexutil.Uint64(h.GasUsed),
+		Root:            h.Root,
+		TxHash:          h.TxHash,
+		ParentHash:      h.ParentHash,
+		UncleHash:       types.EmptyUncleHash,
+		Time:            hexutil.Uint64(h.Time.Unix()),
+		TimeNano:        hexutil.Uint64(h.Time),
+		BaseFee:         (*hexutil.Big)(h.BaseFee),
+		Difficulty:      new(hexutil.Big),
+		PrevRandao:      h.PrevRandao,
+		TotalDiff:       new(hexutil.Big),
+		Hash:            &h.Hash,
+		Epoch:           hexutil.Uint64(hash.Event(h.Hash).Epoch()),
+		WithdrawalsHash: &types.EmptyWithdrawalsHash,
```

Comment:
I think this field needs to be `nil` if the `Sonic` update is not active, otherwise we are breaking backward compatibility with the Fantom network, which is required for running some of the release tests.

An example of such a test is in line 83:
```
    prevRandao := common.Hash{}
	if rules.Upgrades.Sonic {
		prevRandao.SetBytes([]byte{1}) // TODO provide pseudorandom data?
	}
```